clang-tools-extra/docs/clang-tidy/checks/readability/use-explicit-namespaces.rst

```diff
@@ -0,0 +1,24 @@
+.. title:: clang-tidy - readability-use-explicit-namespaces
+
+readability-use-explicit-namespaces
+===================================
+
+This check detects and fixes references to members of namespaces where the namespace is not explicity specified in the reference.  By default, this will only change references that are found through a using namespace directive.
```

Comment:
Please synchronize first statement with Release Notes. `This check` is omitted. Please follow 80 characters limit. Double space before `This check`.